pages/Documents/Reporting/DataUsageAuditAPI/Methods/retrieve-data-usage-logs.md

```diff
@@ -24,7 +24,7 @@ All auditing logs are returned in a JSON format, aggregated per account, API typ
 | :-----    | :--------------- | :-------------- | :--- |
 | from | The starting time (in epoch time) from which the data will be filtered.| numeric | required |
 | to | The end time (in epoch time) until which the data will be filtered. Max time range supported is one month | numeric | required |
-| agentId | If the data was accessed from Conversational Cloud's UI, this is the agentId of the user who did so. In case the data wa accessed using a historical data API, this is the AppKey used <br> Example: agentId=4153. To retrieve all user 4153 audit logs, use agentId=all for all. | numeric | required |
+| agentId | If the data was accessed from Conversational Cloud's UI, this is the agentId of the user who did so. In case the data was accessed using a historical data API, this is the AppKey used <br> Example: agentId=4153. To retrieve all user 4153 audit logs, use agentId=all for all. | numeric | required |
```

Comment:
from Conversational Cloud's UI >> did we decide on adding **the** before Conversational Cloud? If so, it can be added here. I think it can be added here either way because it is referring to the UI.

Reply:
No, we didn’t resolve this yet. Leaving as is until there is alignment.